app/actors/sufia/grant_edit_to_depositor_actor.rb

```diff
@@ -0,0 +1,15 @@
+module Sufia
+  class GrantEditToDepositorActor < CurationConcerns::Actors::GrantEditToDepositorActor
+    private
+
+      # Overriding the parent class so it doesn't grant access
+      # if mediated deposit is enabled
+      def grant_edit_access
+        super unless mediated_deposit?
```

Comment:
So if `mediated_deposit?` returns `false` will that halt the stack because `#grant_edit_access` returns `nil`?